code/addons/onboarding/src/App.tsx

```diff
@@ -134,7 +142,12 @@ export default function App({ api }: { api: API }) {
           codeSnippets={codeSnippets}
           addonsStore={addons}
           onFinish={() => {
-            api.selectStory('example-button--warning');
+            try {
+              api.selectStory('example-button--primary');
```

Comment:
Shouldn't the selected story still be `warning`? And can you quickly elaborate, on why the safety catches are necessary or in which particular scenarios `selectStory` throws?